server/src/schema.js

```diff
@@ -24,10 +34,19 @@ type Mutation {
 }
 
 type Subscription {
-  taskCreated: Task,
-  taskModified: Task,
-  taskDeleted: ID
+  taskMutated: TaskMutationPayload!
 }
+
+type TaskMutationPayload {
+  mutation: MutationType!
+  task: Task!
+}
+enum MutationType {
```

Comment:
Does this result in numbers sent or strings?

Reply:
because I have defined types on the client too its a string